.github/renovate.json5

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base", "schedule:weekly", "group:allNonMajor"],
+  "labels": ["dependencies"],
+  "rangeStrategy": "bump",
```

Comment:
This line will bump the dependency range in `package.json`. I think it's nice, but if we don't want to and prefer to update the lockfile only like before, it's also possible by removing this line.

Reply:
I would agree this would be best practices to me since the lock file is going to get updated anyways from someone building the library